.github/workflows/pyside6.yml

```diff
@@ -0,0 +1,34 @@
+name: Run PySide6 Tests
+
+on:
+  push:
+    branches:
+      - "*"
+  pull_request:
+    branches:
+      - "*"
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - os: ubuntu-latest
+            VFXPLATFORM: "2018"
+            PYTHON: "3.6"
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: pip install PySide6
+        run: |
+          pip install PySide6
```

Comment:
## Choose `pip` explicitly, install needed dependencies.

As @mottosso mentioned, these tests will not be run within the Docker container anymore. So you probably also need to **install** a few more **dependency packages** here. 

You'll probably need to install`nose` and `nosepipe`, maybe a few more. (_→ You can have a look at the [Dockerfile](https://github.com/mottosso/Qt.py/blob/master/Dockerfile.vfxplatform2018#L452), to see what's needed._)

---

Also, it might be good to **be explicit** about the **pip/python version** that you are using.

**So please replace:**
_(Using `Python 3.10` in my example...)_
```bash
pip install PySide6
```
with:
```bash
python3.10 -m pip install PySide6
```